src/dashboard/tests/test_archival_storage.py

```diff
@@ -207,7 +207,7 @@ def setUp(self):
         self.client = Client()
         self.client.login(username="test", password="test")
         helpers.set_setting("dashboard_uuid", "test-uuid")
-        self.data = '{"time":1586880124134,"columns":[{"visible":true},{"visible":true},{"visible":true},{"visible":false},{"visible":false},{"visible":true},{"visible":false},{"visible":true}]}'
+        self.data = '{"time":1588609847900,"columns":[{"visible":true},{"visible":true},{"visible":false},{"visible":true},{"visible":false},{"visible":false},{"visible":true},{"visible":true},{"visible":false},{"visible":true}]}'
```

Comment:
Thanks for this context! I moved the archival storage tests to `src/dashboard/tests` because with the exceptions of the `administration` and `archival_storage` components (until this change), the tests for all of the components live at `src/dashboard/tests` and I found it confusing to have dashboard tests and fixtures scattered across several locations.

The way I'm reading the linked PR, I think this change is in-line with that discussion recommendation, as it puts all the tests for the dashboard app together. This comment for example references `src/dashboard/tests` as an expected location for dashboard tests: https://github.com/artefactual/archivematica/pull/1448#issuecomment-510458475.

There is definitely some ambiguity around the word "component", which I think may be referring in some cases in this conversation to the Django apps (`dashboard`, `MCPServer`, `MCPClient`) rather than the dashboard components. However it's also very possible that I'm misunderstanding! What do you think?

Reply:
I feel the conversation could be clearer for sure. My understanding is different. I think the ultimate conclusion is to have tests per component.

My reference point given there is doubt would be the history of the modules themselves, and for each:

* [Administration](https://github.com/artefactual/archivematica/commits/stable/1.11.x/src/dashboard/src/components/administration/tests).
* [Archival storage](https://github.com/artefactual/archivematica/commits/stable/1.11.x/src/dashboard/src/components/archival_storage/tests).

The tests only landed there this last tranche of development for `1.11`. e.g. [Storage service enhancements](https://github.com/artefactual/archivematica/commit/0410e5d0639222c4e46244a9c46291267895f881#diff-a6384982158121f309654f1f7166fd44) which was connected to the previous discussion.

I would stick with that consistency for now and move the archival storage test back. That's not to say it can't be resolved in this PR, but perhaps separating concerns, it could be logged as a new issue to be discussed. And we can tackle it in a future issue. It seems like maybe the strategy does need encoding somewhere? From a change/commit/PR perspective, then doing this in a future issue would give us each something to reference that says, doing (x) because of (y). Does that sound okay for now?

Reply:
Yes, sounds good for now. Thanks Ross!